src/services/BackendService/BackendService.ts

```diff
@@ -23,6 +23,16 @@ const TRANSMISSION_RISK_LEVEL = 1;
 // See https://github.com/cds-snc/covid-shield-server/pull/176
 const LAST_14_DAYS_PERIOD = '00000';
 
+const CONTAGIOUS_DAYS_BEFORE_SYMPTOM_ONSET = 2;
+// todo: fix this - not sure this should be 11 days
+const CONTAGIOUS_DAYS_BEFORE_TEST_DATE = 11;
+
+// todo: import this from BaseTekUploadView when PR #1088 is merged
+interface ContagiousDateInfo {
+  dateType: 'symptomOnsetDate' | 'testDate' | 'noDate';
```

Comment:
It would be great to use an Enum for `dateType`. That way there is no need to compare against a string later on in the code.

See `ExposureStatusType` in **ExposureNotificationService.ts** as an example.